src/components/BeaconsTable.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import { testBeacons } from "../gateways/BeaconsGateway.testData";
+import { IBeaconsGateway } from "../gateways/IBeaconsGateway";
+import { BeaconsTable } from "./BeaconsTable";
+
+describe("<BeaconsTable>", () => {
+  let beaconsGatewayDouble: IBeaconsGateway;
```

Comment:
I was able to test everything I felt I needed to using `testing-library` as unit tests, so I didn't feel the need to use Cypress yet.